src/rust/ide/view/src/searcher/icons.rs

```diff
@@ -51,6 +51,51 @@ fn arrow(length:f32,width:f32,head_length:f32,head_width:f32) -> AnyShape {
     (line + head).into()
 }
 
+fn grid(stroke_width:f32,cell_size:f32) -> AnyShape {
+    let horizontal = HalfPlane();
+    let horizontal = horizontal.translate_y(stroke_width.px()) - horizontal;
+    let vertical   = HalfPlane().rotate((PI/2.0).radians());
+    let vertical   = vertical.translate_x(stroke_width.px()) - vertical;
+    (horizontal + vertical).repeat((cell_size.px(),cell_size.px())).into()
+}
```

Comment:
All of these should have docs. TBH I do not know what "grid" or "arc" do. They can be many things